### ISSUE TITLE
Left menu: create "Partnerships" button/page/section

### ISSUE THREAD

**Issue by elimisteve** [PursuanceProject/ensue#145]
Possible icons:

- FaCodeFork
- FaCompress
- FaConnectdevelop

**Comment by v-stickykeys** [comment]
what do you think of the name "Universe" instead?

**Comment by elimisteve** [comment]
@thevaleriemack Hmm, that _does_ sound super cool, though we want people to know that they can view existing and create new partnerships there...

I know: when we add a JS walkthrough we can say that this is where they can explore the Pursuance universe and where they can propose new partnerships, and eventually we'll let people explore pursuances _on other servers_ that their current server is peered with, in which case it really _will_ be the universe of pursuances -- and in a familiar location that need not change once federation is added.

So, in summary: good idea; let's to it :-) .

**Comment by v-stickykeys** [comment]
@elimisteve https://github.com/thevaleriemack/pursuance/tree/about_pursuance_tab

**Comment by elimisteve** [comment]
@colinfruit Maybe one of these icons?

* FaConnectdevelop
* FaGlobe

**Comment by elimisteve** [comment]
Something vast and epic and formidable.
